### PR TITLE
Mark config fixtures as deprecated

### PR DESCRIPTION
**Pull request by XxXgeoXxX**
### Description (*)
Original issue https://github.com/magento/graphql-ce/issues/794
 - Fixtures with reference #167 to are marked as deprecated.
 - Checked tests.
### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Please refer to the original issue.
AC is following:
- Fixtures with reference #167 to are marked as deprecated (TODOs should be removed)
- Explanatory comment about @magentoConfigFixture annotation is added (after the deprecated tag).
So it should look like `@deprecated use @magentoConfigFixture instead`.
- No tests are using deprecated fixtures.

File: dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/enable_offline_shipping_methods.php [hunk 1]
```diff
@@ -3,6 +3,12 @@
  * Copyright © Magento, Inc. All rights reserved.
```
Comment (lenaorobei) [inline dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/enable_offline_shipping_methods.php:3]: Found the use of this file in `\Magento\GraphQl\AuthorizenetAcceptjs\Customer\SetPaymentMethodTest::testPlaceOrder`. It needs to be replaced with `@magentoConfigFixture`.

File: dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/disable_all_active_payment_methods.php [hunk 2]
```diff
@@ -3,6 +3,15 @@
...
+/**
+ * @deprecated use next @magentoConfigFixture instead:
+ * @magentoConfigFixture default_store payment/fake_vault/active 0
+ * @magentoConfigFixture default_store payment/paypal_billing_agreement/active 0
+ * @magentoConfigFixture default_store payment/fake/active 0
+ * @magentoConfigFixture default_store payment/checkmo/active 0
+ * @magentoConfigFixture default_store payment/free/active 0
+ *
+ */
 // TODO: Should be removed in scope of https://github.com/magento/graphql-ce/issues/167
```
Comment (lenaorobei) [inline dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/disable_all_active_payment_methods.php]: All `TODOs` should be removed.

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Please remove all `TODOs`, check all fixtures occurrences in tests and replace with config fixtures.

File: dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/disable_all_active_payment_methods.php (hunk 2 above)
Comment (XxXgeoXxX) [reply dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/disable_all_active_payment_methods.php]: Done

File: dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/enable_offline_shipping_methods.php (hunk 1 above)
Comment (XxXgeoXxX) [reply dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/enable_offline_shipping_methods.php:3]: Done

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5547](https://jira.corp.magento.com/browse/ENGCOM-5547) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Pull request merged**